concrete/single_pages/dashboard/system/attributes/topics/view.php

```diff
@@ -10,53 +10,68 @@
+                        <option value="<?=$stree->getTreeID(); ?>" <?php if ($tree->getTreeID() == $stree->getTreeID()) {
+        ?>selected<?php
+    } ?>><?=$stree->getTreeDisplayName(); ?></option>
+                    <?php
 } ?>
                 </select>
             </li>
         </ul>
         <ul class="nav navbar-nav navbar-right">
             <li class="navbar-form">
-            <?php if (PermissionKey::getByHandle('remove_topic_tree')->validate() && is_object($tree)) {
-    ?>
-                <button type="button" data-dialog="delete-topic-tree" class="btn btn-danger btn-sm"><?=t('Delete Topic Tree')?></button>
-            <?php 
-} ?>
-            <?php if (PermissionKey::getByHandle('add_topic_tree')->validate()) {
-    ?>
-                <button onclick="window.location.href='<?=$view->url('/dashboard/system/attributes/topics/add')?>'" class="btn btn-default btn-sm"><?=t('Add Topic Tree')?></button>
-            <?php 
-} ?>
+                <?php if (PermissionKey::getByHandle('add_topic_tree')->validate()) {
+        ?>
+                    <button onclick="window.location.href='<?=$view->url('/dashboard/system/attributes/topics/add'); ?>'" class="btn btn-primary btn-sm"><?=t('Add Topic Tree'); ?></button>
+                    <?php
+    } ?>
+                <?php if (PermissionKey::getByHandle('edit_topic_tree')->validate() && is_object($tree)) {
+        ?>
+                    <button type="button" data-dialog="edit-topic-tree" class="btn btn-default btn-sm"><?=t('Edit Topic Tree'); ?></button>
+                    <?php
+    }
+                ?>
+                <?php if (PermissionKey::getByHandle('remove_topic_tree')->validate() && is_object($tree)) {
+                    ?>
+                    <button type="button" data-dialog="delete-topic-tree" class="btn btn-danger btn-sm"><?=t('Delete Topic Tree'); ?></button>
+                    <?php
+                } ?>
             </li>
         </ul>
     </div>
 </nav>
 
 <?php if (is_object($tree)) {
-    ?>
-	<div data-tree="<?=$tree->getTreeID()?>">
+                    ?>
+	<div data-tree="<?=$tree->getTreeID(); ?>">
 	</div>
 
 	<script type="text/javascript">
 	$(function() {
 		$('select[name=topicTreeIDSelect]').on('change', function() {
-			window.location.href = '<?=$view->url('/dashboard/system/attributes/topics', 'view')?>' + $(this).val();
+			window.location.href = '<?=$view->url('/dashboard/system/attributes/topics', 'view'); ?>' + $(this).val();
 		});
 		
 		$('[data-tree]').concreteTree({
-			'treeID': '<?=$tree->getTreeID()?>'
+			'treeID': '<?=$tree->getTreeID(); ?>'
 		});
 
         $('button[data-dialog=delete-topic-tree]').on('click', function() {
             jQuery.fn.dialog.open({
                 element: '#ccm-dialog-delete-topic-tree',
                 modal: true,
                 width: 320,
-                title: <?=$jh->encode(t("Delete Topic Tree"))?>,
+                title: <?=$jh->encode(t("Delete Topic Tree")); ?>,
```

Comment:
The JSON helper is deprecated: we can now use the `json_encode` directly (so: `title: <?=json_encode(t("Delete Topic Tree")); ?>,`